addons/point_of_sale/models/res_partner.py

```diff
@@ -12,6 +12,12 @@ class ResPartner(models.Model):
         groups="point_of_sale.group_pos_user",
     )
     pos_order_ids = fields.One2many('pos.order', 'partner_id', readonly=True)
+    pos_contact_address = fields.Char('PoS Address', compute='_compute_pos_contact_address')
+
+    @api.depends(lambda self: self._display_address_depends())
+    def _compute_pos_contact_address(self):
+        for partner in self:
+            partner.pos_contact_address = partner._display_address(without_company=True)
```

Comment:
```suggestion
    @api.depends(lambda self: self._display_address_depends())
    def _compute_pos_contact_address(self):
        for partner in self:
            partner.pos_contact_address = partner._display_address(without_company=True)
```
```suggestion
    def _compute_contact_address(self):
        super()._compute_contact_address()
        for partner in self:
            partner.pos_contact_address = partner._display_address(without_company=True)
```

override this `_compute_contact_address`